content/docs/details/how-roadie-connects/index.md

```diff
@@ -0,0 +1,143 @@
+---
+title: How Roadie connects to your tools
+publishedDate: '2023-05-04T13:00:00.0Z'
+description: Roadie Backstage requires connectivity into your software delivery tools. Here's how we accomplish that easily and securely.
+---
+
+## Introduction
+
+In order to add the most value, Backstage requires connections into most of the software used in your delivery pipeline.
+
+Roadie is a SaaS provider of Backstage, so we’ve invested deeply into providing simple and secure ways to connect our platform to your tools, regardless of where they are hosted.
+
+There are multiple ways to establish connections, including Oauth2, proxying, and by using a broker service. This article explains each option, and lays out the situations in which each would be used.
+
+## Connecting to SaaS software
+
+Many Backstage plugins connect from Roadie to a third-party SaaS solution (think of PagerDuty or Datadog) to display data inside the Backstage developer portal.
+
+The two best ways to establish a connection are Oauth2 and proxying.
+
+### Option 1: Oauth2
+
+With Oauth2, the user makes a direct connection from the plugin running in their browser, to the APIs of the SaaS solution.
+
+For example, the [Google Calendar plugin](/docs/integrations/gcalendar/) uses an Oauth2 connection to communicate with the Google APIs. A user who has not authenticated will see a prompt asking them to sign in.
+
+![Google Calendar Backstage plugin asking the user to sign in to Google](./google-calendar-oauth2-prompt.png)
+
+Clicking “SIGN IN” will open a dialog box which asks the user which Google account they want to sign in with. Once the user accepts the prompts, the authentication handshake is completed, and their calendar appears inside Roadie.
+
+This connection is between Google and the user’s browser directly. No information about the user’s calendar is shared with Roadie during this process.
+
+### Option 2: The Proxy
+
+The proxy runs on the Roadie backend. It accepts incoming requests, modifies them — usually to add authentication headers — and forwards them on to a third-party SaaS API. 
+
+The proxy code lives in the [proxy-backend plugin](https://github.com/backstage/backstage/tree/master/plugins/proxy-backend) in the Backstage OSS project. Under the hood, Backstage delegates proxying responsibilities to the popular and battle-tested [http-proxy-middleware](https://github.com/chimurai/http-proxy-middleware) package.
+
+The [Backstage PagerDuty plugin](/docs/integrations/pagerduty/) is an example of a plugin which relies on the proxy to connect to the PagerDuty APIs.
+
+Roadie users must obtain an API token for the PagerDuty APIs. They then store this token in Roadie’s secrets page. These tokens are stored in AWS’s Parameter Store product, using the `SecureString` type. SecureString objects are encrypted with a per-tenant KMS key.
```

Comment:
Mentioning `SecureString` might be too much technical detail. explaining encryption with tenant specific keys should be enough